.github/workflows/sanity.yml

```diff
@@ -138,8 +163,7 @@ jobs:
 
       - name: Load image artifact
         run: |
-          docker load -i /tmp/ramen-operator.tar
-          kind load docker-image ${IMAGE_TAG_BASE}-operator:${IMAGE_TAG} --name ${KIND_CLUSTER_NAME}
+          kind load image-archive /tmp/ramen-operator.tar --name ${KIND_CLUSTER_NAME}
```

Comment:
(nit) Why in this one instance it is `kind` and not `${{env.DOCKERCMD}}`?

Reply:
Well, I just deleted the docker line above. Earlier, it was a 2 step process where docker loaded the tar into its images and then kind loaded the image from docker daemon to the the cluster.

If we are using podman, with the version of kind we are using, it is not possible to use the `kind load docker-image` variant(it requires docker daemon and podman isn't sufficient).

Instead, we load the image into the cluster in a single step by using the `kind load image-archive` command that takes the tar variant directly.